src/usr/local/www/diag_packet_capture.php

```diff
@@ -492,6 +496,13 @@ function hostport_array_fixer($items,$type) {
 		'fa-download'
 	))->addClass('btn-primary');
 
+	$form->addGlobal(new Form_Button(
+		'clearbtn',
+		'Clear Log',
```

Comment:
This isn't a log, it's the previous packet capture. It would be more correct to label this `Clear Last Capture`.

Reply:
I used that at first, but the button gets too large esthetically...
What about a shorter `Clear Capture` to match the the existing buttons `View Capture` and `Download Capture` ?
I push a separate commit so that you can decide up to where you want to merge it

Reply:
Sure, that's reasonable.